digits/extensions/view/imageSegmentation/view.py

```diff
@@ -6,6 +6,8 @@
 import matplotlib as mpl
 import numpy as np
 import PIL.Image
+import json
+import skfmm
```

Comment:
According to PEP8, I need to add a blank line. (https://www.python.org/dev/peps/pep-0008/#imports). Are you saying @lukeyeager prefers third party imports between application and local imports?